src/commons/sagas/WorkspaceSaga.ts

```diff
@@ -85,6 +90,15 @@ let breakpoints: string[] = [];
 export default function* WorkspaceSaga(): SagaIterator {
   let context: Context;
 
+  yield takeEvery(
+    ADD_HTML_CONSOLE_ERROR,
+    function* (action: ReturnType<typeof actions.addHtmlConsoleError>): any {
```

Comment:
```suggestion
    function* (action: ReturnType<typeof actions.addHtmlConsoleError>) {
```

Maybe we should let TypeScript infer the type automatically instead of forcing it to be `any`?